src/content/tutorials/fleet-management/cluster-management/node-pools/index.md

```diff
@@ -71,6 +71,97 @@ worker  mycluster-pool0  ip-10-1-6-225.eu-central-1.compute.internal
 ```
 
+{{< /tab >}}
+{{< tab id="nodepool-capa-karpenter-config" for-impl="capa_ec2_karpenter" >}}
+
+In your workload cluster values, you can specify the node pool configuration to use `karpenter`
+
+```yaml
+global:
+  metadata:
+    name: test-cluster
+    organization: giantswarm
+  nodePools:
+    pool0:
+      type: karpenter # This is the key difference that will make karpenter manage your worker nodes
+      consolidateAfter: 6h
+      consolidationPolicy: WhenEmptyOrUnderutilized
+      consolidationBudgets: # You can control when and how nodes will be updated / rolled with the budgets
+        - nodes: "20%"
+        - schedule: "0 9 * * mon-fri"
+          duration: "8h"
+          nodes: "0"
+          reasons:
+            - "Drifted"
+      customNodeLabels:
+        - yourcustomlabel=canbeconfiguredhere
+      customNodeTaints:
+        - key: example.com/special-taint
+          effect: NoSchedule
+      expireAfter: 20h # The amount of time a Node can live on the cluster before being deleted by Karpenter
+      requirements: # In the 'requirements' you can specify which nodes you want karpenter to consider or to ignore for your node pool
+        - key: karpenter.k8s.aws/instance-cpu
+          operator: In
+          values:
+            - "4"
+            - "8"
+            - "16"
+            - "32"
+        - key: karpenter.k8s.aws/instance-hypervisor
+          operator: In
+          values:
+            - nitro
+        - key: kubernetes.io/arch
+          operator: In
+          values:
+            - amd64
+        - key: karpenter.sh/capacity-type
+          operator: In
+          values:
+            - spot
+            - on-demand
+        - key: kubernetes.io/os
+          operator: In
+          values:
+            - linux
+      terminationGracePeriod: 30m # The amount of time a Node can be draining before Karpenter forcibly cleans up the node
+      limits: # Maximum amount of resources that the node pool can consume
+        cpu: 1000
+        memory: 1000Gi
+  providerSpecific:
+    region: "eu-west-1"
+  release:
+    version: 33.0.0
+```
+
+A node pool is identified by a name that you can pick as a cluster administrator. The name must follow these rules:
+
+- must be between 5 and 20 characters long
+- must start with a lowercase letter or number
+- must end with a lowercase letter or number
+- must contain only lowercase letters, numbers, and dashes
+
+For example, `pool0`, `group-1` are valid node pool names.
+
+The node pool name will be a suffix of the cluster name. In the example above, the node pool name will be `mycluster-pool0`.
+
+All nodes in the node pool will be labeled with the node pool name, using the `giantswarm.io/machine-pool` label. You can identify the nodes' node pool using that label.
+
+When using `type: karpenter` in your node pool configuration, `karpenter` will be deployed in your cluster to manage your worker nodes.
```

Comment:
The name Karpenter, like any other tool name, should not be formatted as a code literal. Please just spell it as Karpenter and add it to the vocabulary to avoid vale complaints.

Reply:
Done, please take a look.